packages/flutter_markdown/test/table_test.dart

```diff
@@ -121,6 +121,27 @@ void defineTests() {
       },
     );
 
+    testWidgets(
+      'table cell vertical alignment should follow stylesheet',
```

Comment:
Isn't this only testing one value? If so, then everything would pass even if someone changed it to just always use `top` the way it used to always use `middle`.

You also don't have any testing of the default handling, unless I missed it.